modules/client-configuration/pages/supported-features-sles.adoc

```diff
@@ -1,8 +1,8 @@
 [[supported-features-sles]]
-= Supported {slsa} and {opensuse} Features
+= Supported {slsa} and {opensuse} Leap Features
```

Comment:
On the left menu we mention this as Leap, so hot having it here is a mistake. openSUSE Leap Micro is documented separately.